instrumentation/spring/spring-cloud-gateway/spring-cloud-gateway-2.0/javaagent/build.gradle.kts

```diff
@@ -7,6 +7,7 @@ muzzle {
     group.set("org.springframework.cloud")
     module.set("spring-cloud-starter-gateway")
     versions.set("[2.0.0.RELEASE,]")
+    assertInverse.set(true)
```

Comment:
Is the reason for this change related to https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/12314 ?

Reply:
> Is the reason for this change related to #12314 ?

It seems that the author forgot to add assertInverse before.